include/entity/scene.h

```diff
@@ -0,0 +1,27 @@
+#ifndef ENTITY_SCENE_H
+#define ENTITY_SCENE_H
+
+#include <vector>
+#include <entity/light.h>
+#include <graphic/color.h>
+
+class Scene {
+private:
+    std::vector<Light> lights;
+    Color environmentColor; // QUESTION: Should this be a Color or a Pixel?
+    // TO DO: Add objects
+public:
+    Scene();
+    Scene(std::vector<Light> lights, Color environmentColor);
+
+    std::vector<Light> getLights() const;
+    Color getEnvironmentColor() const;
+
+    void setLights(std::vector<Light> lights);
+    void setEnvironmentColor(Color environmentColor);
+
+    //TO DO: Add objects getters and setters
+    //TO DO: Add intersection method
```

Comment:
Marcando aqui +2 to do para não esquecer

Reply:
@kinhosz se tiver alguma ideia em relação ao método de interseção (`intersect`), lança a braba para eu já ir pensando a respeito :eyes:

Reply:
vamos criar o `intersect` em outro PR. Vou abrir uma issue com os detalhes dela, pq ela vai ser bem mais complexa e acho que dentro desse escopo acaba ficando muita coisa.

Reply:
sobre os getters e os setters, pode ficar a vontade. Só queria abrir a discussão para uma coisa:

`vale a pena ter todos os atributos da classe disponíveis para ser setados ou pegados?`

Por exemplo, o plane, faz sentido a gente setar um vetor normal? o vetor normal não deveria ser constante durante toda a vida de um plane? 

Fica essas dúvidas

Reply:
@kinhosz vamos abrir uma issue para o `intersect`, posso tocar isso depois que esse PR #64 subir para `main`, com a estrutura base do `Scene`.

Reply:
Em relação a essa questão dos getters/setters, realmente, dependendo da classe alguns atributos não fazem sentidos serem setados ou até mesmo "solicitado". A gente pode levantar o que não faz sentido ter getter/setter e depois, ajustar tudo em um único PR, o que achas?